### PR TITLE
✨ Transfer contract ownership restructure

### PR DESCRIPTION
**Pull request by SurgeCode**
-Transfer Contract Ownership restructure
-Unit tests
-Docs
-Integration Tests

### CODE REVIEW

File: packages/sdk/src/transferContractOwnership/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    //intended owner of the contract
+    nextOwner: string;
+    options?: {
+        //keep the current minters, defaults to true if options aren't given
+      keepMinters: boolean;
+    };
+  };
+```
+
+Example usage of transferContractOwnership method in a hypothetical React component:
```
Comment (tifrel) [inline packages/sdk/src/transferContractOwnership/README.md:26]: Shouldn't `transferContractOwnership` be marked as code?

File: packages/sdk/src/transferContractOwnership/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+{% code title="transferContractOwnershipUI.ts" overflow="wrap" lineNumbers="true" %}
+
+```typescript
+import { useState } from 'react';
+import { useWallet } from '@mintbase-js/react';
+import { execute, transferContractOwnership } from '@mintbase-js/sdk';
+
+
+export const transferContractOwnershipUI = ({ nextOwner, contractId }:any) => {
```
Comment (tifrel) [inline packages/sdk/src/transferContractOwnership/README.md:35]: Missing space after colon. Does ESLint not work for code embedded in MD?

Comment (cif) [reply packages/sdk/src/transferContractOwnership/README.md:35]: It does not unfortunately, I think after this effort we'll want to take two full passes at documentation.. one right before alpha release prior to merging pr in gitbook-docs repo and then one before we release beta

**Review by tifrel — APPROVED** [review]

**Review by sainthiago — APPROVED** [review]

File: packages/sdk/src/transferContractOwnership/README.md (hunk 2 above)
Comment (SurgeCode) [reply packages/sdk/src/transferContractOwnership/README.md]: Added it in, lets review this more thoroughly before release like you said

**Pull request merged**